ehsm_kms_service/delete_secret_thread.js

```diff
@@ -0,0 +1,118 @@
+const {
+    base64_decode
+} = require('./function')
+
+// Set a queue to record the waiting time
+var delete_task_queue = {}
```

Comment:
What would happen if I use forcedelete API deleted a secret already in delete_task_queue? The secret would be deleted twice?

Reply:
forcibly deleted content will be deleted directly when running secret delete api, and will not be added to delete_task_queue

Reply:
I mean the secret is planned to be deleted, for example, 8 hours, at first.
But I use force delete when there are 2 hours left and has been added in delete_task_queue.
What would happen?

Reply:
This is a problem, it will be modified to after force deletion, delete this delete_task